Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM alpine:latest
 RUN apk --update add ca-certificates \
                      mailcap \
-                     curl
+                     curl \
+                     jq
 
 HEALTHCHECK --start-period=2s --interval=5s --timeout=3s \
-  CMD curl -f http://localhost/health || exit 1
+  CMD curl -f http://localhost:$(jq '.port' /.filebrowser.json)/health || curl -f http://localhost/health || exit 1
```

Comment:
```suggestion
  CMD curl -f http://localhost:$(jq '.port' /.filebrowser.json || 80)/health || exit 1
```

Reply:
As I explained it in my last comment, it's  impossible to do it like this: if the json node `port` does not exist, the `jq` command will still return something (the `null` string) with a `0` exit code. The `|| 80` won't work.

The only thing we could do here would be `curl -f http://localhost:$(jq '.port' /.filebrowser.json || echo '80')/health`, but this will only catch a `jq` error, as a file not found.